turtlebot_actions/action/TurtlebotMove.action

```diff
@@ -1,6 +1,7 @@
 #goal definition
 float32 turn_distance     # in radians, ccw = +, cw = -
 float32 forward_distance  # in meters, forward = +, backward = -
+float32 vel_rate          # Scale, between 0.1 to 1. Enable the vel_control mode in the turtblebot_move_action_server
```

Comment:
This should be a real value m/s not an abstract scale. And the name vel_rate is a little ambiguous. Is it expected to be the velocity at which it travels?